extern_test.rs

```diff
@@ -0,0 +1,13 @@
+extern crate creusot_contracts;
+use creusot_contracts::*;
+
+#[creusot::no_translate]
+#[creusot::extern_spec]
+#[requires(true === true)]
+fn extern_xxxx<T>() -> Vec<T> {
```

Comment:
I don't understand the syntax here. Is it supposed to be the spec of function `xxxx`? Why does this function have a body?

Reply:
`xxxx` represents a random uuid. We attach the spec to `extern_xxxx` and inside have a call to the function we really want to specify. 

This structure is partially caused by the fact we can only resolve names after/during type checking. To even identify `Vec::new` we must run the type checker.